Assets/BossRoom/Scripts/Shared/ApplicationController.cs

```diff
@@ -47,11 +47,18 @@ private void Awake()
             scope.BindAsSingle<LocalLobbyUser>();
             scope.BindAsSingle<LocalLobby>();
 
-            //this message channel is essential and persists for the lifetime of the lobby and relay services
+            //these message channels are essential and persist for the lifetime of the lobby and relay services
             scope.BindMessageChannelInstance<UnityServiceErrorMessage>();
-
-            //this message channel is essential and persists for the lifetime of the lobby and relay services
             scope.BindMessageChannelInstance<ConnectStatus>();
+            scope.BindMessageChannelInstance<DoorStateChangedEventMessage>();
+
+            //these message channels are essential and persist for the lifetime of the lobby and relay services
+            //they are networked so that the clients can subscribe to those messages that are published by the server
+            scope.BindNetworkedMessageChannelInstance<LifeStateChangedEventMessage>();
```

Comment:
binding should be done at the state scope? let's investigate, this way no need for disconnect complexity?